### DISCUSSION TITLE
Question about the different 'Day' between test_x and train set

### DISCUSSION THREAD

**Discussion by moonquakes66** [PaddlePaddle/PaddleSpatial#63] [BaiduKDDCup2022] [answered]
I just find that when the value of the columns in {'TurbID','Day',Tmstamp'} in the test_x are the same as in the train set you provided,they don't have the same weather characteristics values such as 'Wspd' or 'Patv', So does it means the 'Day'==1 in the test_x isn't the same day in the train set?(Or the diff is'TurbID' or'Tmstamp'?)

**Comment by ramber1836** [comment, marked as answer]
Please refer to the discussion #39

https://github.com/PaddlePaddle/PaddleSpatial/discussions/39#discussion-4058001

**Reply by hongyingyue** [reply]
@ramber1836 
Hey, due to we have lots of files in test data folder, can we imagine that every test file's time information is correct and is sampled from the same re-start test days?

And during the official test phase and final test phase, the sampling and date information will be provided? like, are there sampled from same period, or the final test phase are sampled from future?

Thanks in advance

**Reply by ramber1836** [reply]
"Hey, due to we have lots of files in test data folder, can we imagine that every test file's time information is correct and is sampled from the same re-start test days? "
A: Yes.

"And during the official test phase and final test phase, the sampling and date information will be provided? like, are there sampled from same period, or the final test phase are sampled from future? "
A: We will consider the option of providing date information carefully.

**Reply by hongyingyue** [reply]
@ramber1836 
Thanks for the fast response. 

For the second question, I fully agree with you. That's also important for participants to choose the final model. Due to the final score just relies on the last submit. My suggestion is at least telling us the relationship between the test phase samples and final phase samples (same period but different samples, or final test dataset is further future period ). Otherwise, the shakeup may come from our assumption about this, instead of the model's general performance.

Of course, any decision from your side is fine for me. Thanks
